pgactivity/Data.py

```diff
@@ -359,6 +361,9 @@ def pg_get_activities(self, duration_mode=1):
     WHERE
         state <> 'idle'
         AND pid <> pg_backend_pid()
+        AND CASE WHEN %(min_duration)s = 0 THEN true
+            ELSE extract(epoch from now() - query_start) > %(min_duration)s
```

Comment:
`query_start` should be replaced by `pg_stat_activity.{duration_column}` since this commit https://github.com/dalibo/pg_activity/commit/c8b78e14e640d975fc57ef4b95e6fe29a7d7f7d8 has introduced variation on duration column.